packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -83,16 +86,41 @@ export interface ITableQuadrantStackProps extends IProps {
     isVerticalScrollDisabled?: boolean;
 
     /**
-     * The number of frozen columns.
+     * A list of `TableLoadingOption`. Loading cells may have different sizes
+     * from potentially custom cells in the header or body, so we need to know
+     * when the loading states change in order to synchronize quadrant sizes
+     * properly.
+     */
+    loadingOptions?: TableLoadingOption[];
+
+    /**
+     * The number of columns. Affects the layout of the table, so we need to
+     * know when this changes change in order to synchronize quadrant sizes
+     * properly.
+     */
+    numColumns?: number;
+
+    /**
+     * The number of frozen columns. Affects the layout of the table, so we need
+     * to know when this changes change in order to synchronize quadrant sizes
+     * properly.
      */
     numFrozenColumns?: number;
 
     /**
-     * The number of frozen rows.
+     * The number of frozen rows. Affects the layout of the table, so we need to
```

Comment:
i appreciate adding this warning to all relevant props. nit: it may be difficult to scan all the comments to find all these props. maybe prefix like "LAYOUT NOTE" or "UPDATE NOTE" 
 or "REQUIRES QUADRANT RESYNC" could help.

Reply:
alternatively, putting all these props into a sub interface.

Reply:
with a sub-interface the docs would even say "inherited from `ITableQuadrantResyncProps`"

Reply:
Ooh cool ideas. Lemme try them.

Reply:
So some resync-inducing props like `isRowHeaderShown` are also used in other logic, meaning it wouldn't make sense to bundle _all_ resync props into a separate interface. I'll add a `REQUIRES QUADRANT RESYNC` note.

Reply:
Done